Source/SpeechToTextV1/SpeechToText.swift

```diff
@@ -413,6 +424,7 @@ public class SpeechToText {
         speakerLabels: Bool? = nil,
         grammarName: String? = nil,
         redaction: Bool? = nil,
+        audioMetrics: Bool? = nil,
```

Comment:
Does this need to move to the end of the parameters?

Reply:
This is okay -- see previous response.